packages/core/admin/server/tests/admin-user.test.e2e.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
-const _ = require('lodash');
+const { omit } = require('lodash/fp');
 const { createStrapiInstance } = require('../../../../../test/helpers/strapi');
 const { createAuthRequest } = require('../../../../../test/helpers/request');
 const { createUtils } = require('../../../../../test/helpers/utils');
 
 const edition = process.env.STRAPI_DISABLE_EE === 'true' ? 'CE' : 'EE';
 
-const omitTimestamps = obj => _.omit(obj, ['updatedAt', 'createdAt']);
+const omitTimestamps = omit(['updatedAt', 'createdAt']);
+const omitRegistrationToken = omit(['registrationToken']);
```

Comment:
Is thi snecessary because you verify the token isn't there in other tests ?

Reply:
Yes, we need to make sure the token is not in the other answers.